script.js

```diff
@@ -408,6 +442,28 @@ function handleData([
  * HANDLE ADDING MAP LAYERS
  *****************************************/
 
+// Styles
+const scoreFillColors = {
+  '1': '#2ca25f',
+  '2': '#99d8c9',
+  '3': '#e5f5f9',
+};
+
+const scoreDescription = {
```

Comment:
just a note to make sure this doesn't conflict with @jimboweb's method of using an array.

Reply:
@river-honer remove this one as well.